tests/integration/src/fluvio-test-options/src/lib.rs

```diff
@@ -0,0 +1,262 @@
+use structopt::StructOpt;
```

Comment:
This entire crate seems like it is revolving around TestOption, when TestOption is really just test parameters for the one single smoke test. I think that the test macro should not need to interact with TestOption at all, we should push TestOption down into the single test case. I believe all of the cluster-related options in TestOption can be represented directly via the `fluvio_cluster::ClusterConfig`.

Reply:
When I abandoned running our tests w/ `cargo test`, I ended up not needing to make up complex ways to pass a test option. 

Tests are now passed a `Fluvio` client and a `TestOption`.

(`TestOption`s are still specialized to the smoke tests at the moment)